src/99-appendices/05-units.md

```diff
@@ -46,67 +46,67 @@ Examples for CMIXF-12 (including the five unicode symbols mentioned above):
-| katal          | kat         | catalytic activity                         |
+| **Unit name**  | **Unit symbol** | **Quantity name**                          |
+| ---------------| --------------- | ------------------------------------------ |
+| metre          | m               | length                                     |
+| kilogram       | kg              | mass                                       |
+| litre (liter)  | L               | volume                                     |
+| second         | s               | time                                       |
+| ampere         | A               | electric current                           |
+| kelvin         | K               | thermodynamic temperature                  |
+| mole           | mol             | amount of substance                        |
+| candela        | cd              | luminous intensity                         |
+| radian         | rad             | angle                                      |
+| steradian      | sr              | solid angle                                |
+| hertz          | Hz              | frequency                                  |
+| newton         | N               | force, weight                              |
+| pascal         | Pa              | pressure, stress                           |
+| joule          | J               | energy, work, heat                         |
+| watt           | W               | power, radiant flux                        |
+| coulomb        | C               | electric charge or quantity of electricity |
+| volt           | V               | voltage (electrical potential), emf        |
+| farad          | F               | capacitance                                |
+| ohm            | Ohm             | resistance, impedance, reactance           |
+| siemens        | S               | electrical conductance                     |
+| weber          | Wb              | magnetic flux                              |
+| tesla          | T               | magnetic flux density                      |
+| henry          | H               | inductance                                 |
+| degree Celsius | oC              | temperature relative to 273.15 K           |
+| lumen          | lm              | luminous flux                              |
+| lux            | lx              | illuminance                                |
+| becquerel      | Bq              | radioactivity (decays per unit time)       |
+| gray           | Gy              | absorbed dose (of ionizing radiation)      |
+| sievert        | Sv              | equivalent dose (of ionizing radiation)    |
+| katal          | kat             | catalytic activity                         |
 
 ## Prefixes
 
 ### Multiples
 
-| Prefix name                                 | Prefix symbol | Factor          |
-| ---------------------------------------------------------- | ---------------------------- | ------------------------------- |
```

Comment:
When adjusting these lines that look weird, please check the PDF build. In most cases, they were made this way to improve the spacing of the table in the PDF.

Reply:
It's true that that used to be a problem (and is a reason why some tables look "weird"), but thanks to https://github.com/bids-standard/bids-specification/pull/560, all tables should be formatted appropriately for the PDF automatically during the build step.